### ISSUE TITLE
Publishing to Bintray/Maven Central

### ISSUE THREAD

**Issue by ddimtirov** [AlloyTools/org.alloytools.alloy#126]
Hello, would you be interested in publishing the Alloy releases to Bintray and Maven Central repositories?

The benefit would be that this would allow more people to take advantage of the tool (for example, in my company we have blanket approval for Maven Central artifacts, but anything else requires procurement)

**Comment by pkriens** [comment]
Good point. I thought [I already did](https://issues.sonatype.org/browse/OSSRH-34048) but see they are not on maven central yet. The snapshots are already there: https://oss.sonatype.org/content/repositories/snapshots/org/alloytools/org.alloytools.alloy.dist/5.1.0-SNAPSHOT/

I need to rebuild for a release but I assume this would give you access?

**Comment by ddimtirov** [comment]
I don't mind rebuilt or new version. As long as it is in Central, and there is no security advisory in the Sonatype's Nexus IQ feed, we should be able to access it.

**Comment by pkriens** [comment]
I need to create a release then, which would be 5.1.1. This  is probably going to take 2 weeks.

**Comment by ddimtirov** [comment]
Thank you!

**Comment by pkriens** [comment]
Snapshots are published to https://oss.sonatype.org/content/repositories/snapshots/org/alloytools/

Somehow the binaries never got pushed on 6.0 release

**Comment by Joshua27** [comment]
Hi,
can you please publish the Alloy 5 and Alloy 6 cores on Maven Central.
Currently, there are only snapshot releases on Sonatype.
Cheers

**Comment by pkriens** [comment]
This has finally been done. Just signed the artifacts, they are on there way to Maven Central

https://central.sonatype.com/search?q=alloytools
